src/services/timer/index.ts

```diff
@@ -7,6 +7,7 @@
 import * as TIMER from './config.js';
 
 import { create } from './service/timer.js';
```

Comment:
Should we also update the export for `timer` here? I see that we only have create defined, but it seems like we should use a similar export for all services.

Reply:
are you saying it should be something like
```
import * as timers from './service/timer.js';
```

and then:
```
export { ...timers, jobs };
```
?

Reply:
The transfer service does `export * as endpoint from './service/endpoint.js'`. I would expect the Timer service to `export * as timer from './service/timer.js` and `export * as job from `./service/job(s?).js`

Reply:
I'm not totally sure, but I think exporting that way here would cause something like this:
```
import { timer } from '@globus/sdk'

...

timer.timer.create
```

Modules are confusing though, so I could easily be wrong about that. If that's true though, it would be a breaking change. 

As far as the directory structure, I'm not sure what the preferred pattern is, looks like we do it both ways so I'm happy to conform to whatever we decide is best, I don't have super strong feelings about it.

Reply:
If you look at how we're using the `timer.create` in the webapp, we're having to `import { timer } from '@globus/sdk'`, instead of `import { timer } from '@globus/sdk/services/timer'`. We're re-exporting `* as timer` in the top-level index in the sdk.

@jbottigliero I don't know what the preferred/intended import path is for consumers.

Reply:
Yeah I think you're right about the exports, in that it's basically equivalent to what we do now, but it does have the added benefit of when we do add methods to the `timers` module, we won't have to manually add them to the export.

Reply:
The idea is that the import paths and naming matches the service endpoints (https://timer.automate.globus.org/redoc); In general we're mimicking what a OpenAPI => REST client might produce.

1. `jobs` is correct based on the implementation there (https://timer.automate.globus.org/redoc#tag/jobs).
2. Updating the `timer` imports/export patterns is something we can do to be more consistent, but let me do that over in #533 to keep this focused on the additions; Breaking change here seems fine since we'll be rolling it out with the `v6` release anyhow.

Reply:
Took this a step further to match the naming changes in the upcoming Python major version Timer => Timer (https://globus-sdk-python.readthedocs.io/en/4.x-dev/upgrading.html#deprecated-timers-aliases-removed).

Which means it'll be something like `timers.timer.create(...)` if your importing from `@globus/sdk` which feels a _a little_ better.